# log type
string type

float32 timeStamp

# log string
string logMessage